example/sofast_fixed/example_measurement_from_image.py

```diff
@@ -0,0 +1,83 @@
+from os.path import join, dirname
+
+import cv2 as cv
+
+import opencsp.app.sofast.lib.image_processing as ip
+from opencsp.app.sofast.lib.MeasurementSofastFixed import MeasurementSofastFixed
+from opencsp.common.lib.geometry.Vxyz import Vxyz
+from opencsp.common.lib.opencsp_path.opencsp_root_path import opencsp_code_dir
+import opencsp.common.lib.render.figure_management as fm
+import opencsp.common.lib.render_control.RenderControlAxis as rca
+import opencsp.common.lib.render_control.RenderControlFigure as rcfg
+import opencsp.common.lib.tool.file_tools as ft
+import opencsp.common.lib.tool.log_tools as lt
+
+
+def example_create_measurement_file_from_image():
+    """Example that creates a SofastFixed measurement file from an image. The image has
+    a point LED light near origin dot.
+    1. Load image
+    2. Define measurement parameters
+    3. Find location of origin point
+    4. Create measurement object
+    5. Save measurement as HDF5 file
+    6. Plot image with annotated origin dot
+    """
+    # General setup
+    # =============
+    dir_save = join(dirname(__file__), 'data/output/measurement_file')
+    ft.create_directories_if_necessary(dir_save)
+    lt.logger(join(dir_save, 'log.txt'), lt.log.INFO)
+
+    # 1. Load image
+    # =============
+    file_meas = join(opencsp_code_dir(), 'test/data/sofast_fixed/data_measurement/measurement_facet.h5')
+    measurement_old = MeasurementSofastFixed.load_from_hdf(file_meas)
+    image = measurement_old.image
```

Comment:
I know it's silly, but it might make this more useful as an example if there were some commented out code for loading the image using opencv or pillow, as if you have an image jpg file instead of a measurement h5 file.

```python
# image = cv2.imread(file_jpg, cv2.IMREAD_GRAYSCALE)
```

Reply:
I like it! Added!